tests/end-to-end/univariate/test_base.py

```diff
@@ -0,0 +1,34 @@
+import numpy as np
+import pytest
+from scipy.stats import norm
+
+from copulas.univariate import Univariate
+
+
+class TestUnivariate():
+
+    @pytest.mark.parametrize('model', Univariate._get_subclasses())
```

Comment:
In other cases parameterizing something like this would make sense.
However, since in this case each model has its own tests module [here](https://github.com/sdv-dev/Copulas/tree/master/tests/end-to-end/univariate) and all of them follow a similar structure, I think it would should add the test within each module, even if the function ends up being identical for each model.

If we consider that there is too much code duplication in these tests, we can open an issue to evaluate the possibility of unifying all the tests in a single tests module in the future, with just a different `setUp` for each one of them.